WFInfo/WFInfo.csproj

```diff
@@ -51,7 +51,7 @@
     <RunCodeAnalysis>false</RunCodeAnalysis>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <PlatformTarget>AnyCPU</PlatformTarget>
+    <PlatformTarget>x64</PlatformTarget>
```

Comment:
Is this going to cause any problems for old cpus?

Reply:
I personally don't think if a person is running Warframe at recommended resolution is still using a 32 bit CPU